### PR TITLE
Issue555 inclusive license

### LINKED ISSUE #555: inclusive license
Harmonize LICENSE with https://www.nist.gov/director/licensing

### PR DESCRIPTION
**Pull request by guyer**
Fixes #555

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/fipy/613)

### CODE REVIEW

**Comment by guyer** [comment]
___
*[fipy/tools/dimensions/physicalField.py, line 44 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS__t2kCxtrNZHw_fCH:-LSaZMdU7fLSEE2kMP7g:bfugauw) ([raw file](https://github.com/usnistgov/fipy/blob/11ac915772967210da59510e218ef352191a0810/fipy/tools/dimensions/physicalField.py#L44)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

This copyright notice should be retained.
</blockquote></details>

I did not retain the copyright notice, as it's in fipy/tools/dimensions/LICENSE, but did restore credit lines for Konrad Hinsen and Greg Ward, as given in the [current version of their source](https://bitbucket.org/khinsen/scientificpython/src/7fef5df489b0588899a6582a3c62ccde447f8ece/Scientific/Physics/PhysicalQuantities.py?at=default&fileviewer=file-view-default#PhysicalQuantities.py-3)

**Comment by guyer** [comment]
___
*[fipy/tools/performance/efficiencyTestHistory.py, line 1 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS_a2hBEpe3SXkzeYZ5:-LSab613DCFut2LgN3It:bk5idk5) ([raw file](https://github.com/usnistgov/fipy/blob/11ac915772967210da59510e218ef352191a0810/fipy/tools/performance/efficiencyTestHistory.py#L1)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

Erm, what's all this about? Perhaps scrub the whole thing?
</blockquote></details>

To keep this pull request atomic, I'm letting this ride, but filed #614 to deal with it

**Comment by guyer** [comment]
___
*[fipy/requirements.txt, line 1 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS_Z70UAzvV50Lz5hiN:-LSac8rf1TDFb-n8raBk:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/f889bb75e4bded34cdfb81c572b572d856efa36e/fipy/requirements.txt#L1)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

This appears to be a new file, which is inconsistent with the apparent intent of this PR. Was it intentionally checked in?
</blockquote></details>

Done.

**Comment by guyer** [comment]
___
*[examples/benchmarking/solvers/.ipynb_checkpoints/analyze-checkpoint.ipynb, line  at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS_WtgTCzp2UnUg-YWI:-LSacBJw8rw21vw1ujJt:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/f889bb75e4bded34cdfb81c572b572d856efa36e/examples/benchmarking/solvers/.ipynb_checkpoints/analyze-checkpoint.ipynb#L)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

IPython Notebook checkpoints definitely do not belong in the repo. Please remove.
</blockquote></details>

Done.

**Comment by guyer** [comment]
___
*[examples/benchmarking/solvers/analyze.ipynb, line 1 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS_WZiY5gvMi4xuisN9:-LSacEqB2ia2enD9hZHX:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/f889bb75e4bded34cdfb81c572b572d856efa36e/examples/benchmarking/solvers/analyze.ipynb#L1)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

This appears to be a new file, which is inconsistent with apparent intent of this PR. Please either document or remove.
</blockquote></details>

Done.

**Comment by guyer** [comment]
___
*[LICENSE.rst, line 2 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS_W7Wb49EpxWBS2QFE:-LSaQ6LdDgZek2jd1ZlC:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/f889bb75e4bded34cdfb81c572b572d856efa36e/LICENSE.rst#L2)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

Exactly duplicates [Director's Statement](https://www.nist.gov/director/copyright-fair-use-and-licensing-statements-srd-data-and-software), with helpful links. LGTM.
</blockquote></details>

Done.

**Comment by guyer** [comment]
___
*[fipy/terms/vanLeerConvectionTerm.py, line 47 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS__Qqo0fTaLKxG97YU:-LSaQ80c0afOC2Hvn3GC:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/11ac915772967210da59510e218ef352191a0810/fipy/terms/vanLeerConvectionTerm.py#L47)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

Empty block-quote should probably also be scrubbed.
</blockquote></details>

Done.

**Comment by guyer** [comment]
___
*[fipy/tools/decorators.py, line 45 at r1](https://reviewable.io/reviews/usnistgov/fipy/613#-LS__dnSEUziNGKrNCrk:-LSaZtKlEMA1YBieqGJr:b-896fix) ([raw file](https://github.com/usnistgov/fipy/blob/11ac915772967210da59510e218ef352191a0810/fipy/tools/decorators.py#L45)):*
<details><summary><i>Previously, tkphd (Trevor Keller) wrote…</i></summary><blockquote>

This copyright notice should be retained.
</blockquote></details>

Done.

File: LICENSE.rst [hunk 1]
```diff
@@ -2,36 +2,31 @@
...
+INJURY WAS SUSTAINED BY PERSONS OR PROPERTY OR OTHERWISE, AND WHETHER OR
+NOT LOSS WAS SUSTAINED FROM, OR AROSE OUT OF THE RESULTS OF, OR USE OF, THE
+SOFTWARE OR SERVICES PROVIDED HEREUNDER.
 
 .. _National Institute of Standards and Technology: http://www.nist.gov/
 .. _NIST: http://www.nist.gov/
-.. _the website: https://www.ctcms.nist.gov/fipy
-.. _this repository: https://github.com/usnistgov/fipy
-.. _title 17 section 105: http://uscode.house.gov/uscode-cgi/fastweb.exe?getdoc+uscview+t17t20+9+0++
+.. _title 17 United States Code Section 105: http://uscode.house.gov/uscode-cgi/fastweb.exe?getdoc+uscview+t17t20+9+0++
```
Comment (tkphd) [inline LICENSE.rst]: Link is broken. Please substitute
https://www.copyright.gov/title17/92chap1.html#105

Comment (tkphd) [reply LICENSE.rst]: ... or, to stay within uscode.house.gov, please use
http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title17-section105&num=0&edition=prelim

Comment (tkphd) [reply LICENSE.rst]: ... or, use the Government Publishing Office (requires specifying the year)
https://www.gpo.gov/fdsys/pkg/USCODE-2017-title17/html/USCODE-2017-title17-chap1-sec105.htm

Comment (guyer) [reply LICENSE.rst]: *sigh* That's not what "tkphd approved these changes 2 days ago" means.

**Pull request merged**